### ISSUE TITLE
Add Feature: Share templates between users/teams/organizations

### ISSUE THREAD

**Issue by ADDAH-temp** [wekan/wekan#2684]
It would be nice to invite/add other users to templates to share them in a team

**Comment by Grumfargh** [comment]
yes i am very interresed by this feature or also can import Card or board into template of current user

**Comment by rubendibattista** [comment]
That would be really great: people invited to a Template board can use the templates defined in that board

**Comment by n8ores** [comment]
Is there any work around to this, having each user have its own templates is a bit painful. We'de like to have one admin user that creates templates and be able to assign those templates to other users as well.

**Comment by xet7** [comment]
@n8ores 

I would think that a real feature for that (like being able to assign template to team or organization) would be better than some workaround.

**Comment by vagnernascimento** [comment]
Hi @xet7, 
When you say "assign template to a team or organization", are you  excluding being able to share directly with members?
Because it seems to me a complementary feature.

Do you think this is too hard to do? Is something the code could be replicate from common boards?
Thanks,

**Comment by xet7** [comment]
From @vagnernascimento 

> Hi @xet7,
> When you say "assign template to a team or organization", are you excluding being able to share directly with members?
> Because it seems to me a complementary feature.

No. There would be ability to add members to template. Maybe sometime later, adding some organization or team would be a way to add more users to same template at once.

> Do you think this is too hard to do? Is something the code could be replicate from common boards?
> Thanks,

Templates feature, boards feature with members etc already exists. I don't think it is too hard. Anyone that has some understanding of some programming language can read enough of Wekan code to see where to make required changes, and then submit pull request.

I would guess, that if it's not yet possible to add new users to template board, then some additional code related to normal board could be added to template board. And then check that template is in use for all members of template board.

For related code, you could search:
```
cd wekan
./find.sh template
```
